packages/react-scripts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skyscanner/backpack-react-scripts",
-  "version": "10.4.0",
+  "version": "10.4.1",
```

Comment:
This should happen after PR is merged and new version is released.

Reply:
Good to know, thanks!